test/CMakeLists.txt

```diff
@@ -90,6 +90,14 @@ add_lit_testsuite(test-bmc "Regression tests for BMC"
   DEPENDS seahorn
   )
 
+add_lit_testsuite(test-gsa "Regression tests for Gated SSA"
+  -v
+  ${CMAKE_CURRENT_SOURCE_DIR}/gsa
+  ARGS
+  --path=${CMAKE_INSTALL_PREFIX}/bin
+  DEPENDS seahorn
```

Comment:
none of the other tests depends on install -- should I add it everywhere in a separate patch?

Reply:
`opsem` does ;) . Sure, if it works for you. While developing, I wanted to use the `test` target instead of `install` so that at every compile my tests run in my compile window. I've discovered that current dependencies only rebuild `seahorn` but don't actually ensure that the new version is used for tests. That's quite annoying to debug. I did not change it everywhere since wasn't sure it works well for all other tests.